workshops/forms.py

```diff
@@ -624,6 +638,13 @@ class SWCEventRequestForm(forms.ModelForm):
                   '-to-admin-fee.html" target="_blank">Look up administration '
                   'fees</a>.',
     )
+    language = selectable.AutoCompleteSelectField(
+        lookup_class=lookups.LanguageLookup,
+        label='Language',
+        required=False,
+        initial=Language.objects.get(name='English'),
```

Comment:
Added a default on the form rather than the model as I couldn't add a FK default.
Is this an issue?